tests/bluetooth/tester/src/btp_bap.c

```diff
@@ -54,9 +55,14 @@ struct audio_stream {
 	struct k_work_delayable audio_clock_work;
 	struct k_work_delayable audio_send_work;
 	uint8_t cig_id;
-	uint8_t cis_id;
 	struct bt_bap_unicast_group **cig;
 	bool already_sent;
+	bool broadcast;
+
+	union {
+		uint8_t cis_id;
+		uint8_t bis_id;
+	};
```

Comment:
The reason to use a `union` here would mainly be for code size purposes. In that case, stuff like `cig`, `cig_id`, `ase_id`, `conn_id` are also only valid for unicast